app/mailers/notify.rb

```diff
@@ -0,0 +1,13 @@
+class Notify < ApplicationMailer
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.notify.monthly_report_registration.subject
+  #
+  def monthly_report_registration(user_id, monthly_report_id)
```

Comment:
__[rails_best_practices]__
* use multipart/alternative as content_type of email [:ref](http://rails-bestpractices.com/posts/2010/08/05/use-multipart-alternative-as-content_type-of-email/)